build/images/base-windows/Dockerfile

```diff
@@ -61,3 +61,33 @@ COPY --from=windows-utility-base ["C:\\\\wins","C:\\\\wins"]
 COPY --from=windows-golang ["C:\\\\Program Files\\\\Go", "C:\\\\Program Files\\\\Go"]
 
 RUN setx /m PATH "C:\Program Files\7-Zip;C:\git\bin;C:\git\usr\bin;C:\mingw64\bin;C:\Program Files\Go\bin;%GOPATH\bin%;%PATH%"
+
+FROM mcr.microsoft.com/windows/servercore:${NANOSERVER_VERSION} as windows-ovs
```

Comment:
Can we use ovs image instead of duplicated script?

Reply:
currently if we try to use ovs image, it will fail because ovs image is not uploaded to registry, we are building two images one is windows-ovs and another is base-windows(which will contain ovs)

Reply:
@wenyingd Do you have more comments for build/images/ovs/Dockerfile.windows? If not we can upload the first ovs image and remove redundant building script in other files to enable CI test.

Reply:
@XinShuYang @wenyingd 
I think for now, we can have the ovs dependencies added to windows base image only and we might not want to create ovs image separately and then add to base image since it will lead to duplicate code unless we upload new ovs image to registry as @XinShuYang mentioned.

As of now for this work we do not need a separate ovs image. So we can remove the file build/images/ovs/Dockerfile.windows, 
An additional work can be taken up later wrt building windows ovs versions which could be automated alongside separate ovs image creation just like linux.

Reply:
Hi @rajnkamr , after discussed with @wenyingd , we believed decoupling ovs image and base image is a must-have feature in long term because in the future we need to support different OS/ovs version for windows image and we don’t want to generate too many base images for different ovs/OS releases. But for this PR you can decide whether to involve this feature. If you want to add ovs build in base image, please raise an issue to track independent windows ovs image support as well.

Reply:
@XinShuYang , Right, For this feature, we can go ahead with adding ovs image to windows base image. Since ovs release is twice a year, we will raise another issue to track decoupling ovs from windows base image as a long term plan.